### PR TITLE
Set up basic form validation and validate required title, author name, and website URL

### PR DESCRIPTION
**Pull request by whitep4nth3r**

### CODE REVIEW

File: pages/create.vue [hunk 1]
```diff
@@ -1,8 +1,25 @@
 <script setup lang="ts">
 // https://vueuse.org/core/useStorage/#usage
+import { Field, Form, ErrorMessage, defineRule } from "vee-validate";
 import { useStorage } from "@vueuse/core";
 import type { Plate } from "../utils/types";
 
+defineRule("required", (value: any, [message]: string) => {
```
Comment (whitep4nth3r) [inline pages/create.vue:7]: I wanted to put these rules in a different file but it was a nightmare.

File: pages/create.vue [hunk 2]
```diff
@@ -11,10 +28,11 @@ function initNewPlate(): Plate {
     user_id: null,
     title: "",
     description: "",
-    author: "",
-    items: [],
-    socialLinks: {
-      website: "",
+    author: {
+      name: "",
+      socialLinks: {
```
Comment (whitep4nth3r) [inline pages/create.vue:33]: This little restructure felt better.

Comment (nikolovlazar) [reply pages/create.vue:33]: Yeah this makes sense!

**Review by nikolovlazar — APPROVED** [review]
Looks good!

**Pull request merged**